docs/troubleshooting/other-troubleshooting-tips/networking.md

```diff
@@ -21,7 +22,7 @@ To test the overlay network, you can launch the following `DaemonSet` definition
 
 :::note
 
-This container [does not support ARM nodes](https://github.com/leodotcloud/swiss-army-knife/issues/18), such as a Raspberry Pi. This will be seen in the pod logs as `exec user process caused: exec format error`.
+The `swiss-army-knife` container does not support Windows nodes. It also [does not support ARM nodes](https://github.com/leodotcloud/swiss-army-knife/issues/18), such as a Raspberry Pi. When the test encounters incompatible nodes, this is recorded in the pod logs with the message, `exec user process caused: exec format error`.
```

Comment:
```suggestion
The `swiss-army-knife` container does not support Windows nodes or [ARM nodes](https://github.com/leodotcloud/swiss-army-knife/issues/18), such as a Raspberry Pi. When the test encounters incompatible nodes, this is recorded in the pod logs with the message, `exec user process caused: exec format error`.
```

Reply:
Seemed like these could be combined here so adjusted the phrasing a bit.

Reply:
I debated doing it this way, but I decided to have separate sentences, because I was concerned that using a conjunction might make it read like a Raspberry Pi is a Windows device.

Reply:
Sounds good!